src/main/js/controls/Scatter/helpers/helpers.js

```diff
@@ -257,7 +257,7 @@ const drawDataPoints = (scale, config, pointGroupPath) => {
                         dataPointActionHandler(value, index, this);
                     },
                     a11yAttributes: {
-                        "aria-hidden": false,
+                        "aria-hidden": document.querySelector(`.${styles.legendItem}[aria-describedby="${value.key}"]`)?.getAttribute('aria-current')==="false",
```

Comment:
Are you using the same `class name` as the selector in all files? Then can we move the `class name` to constant file?

Reply:
The file is which `classNames` are present currently is a constant dictionary. And I think it makes it easier to maintain to have it in a different file than with other constants.